src/main/java/pl/javastart/task/Car.java

```diff
@@ -0,0 +1,18 @@
+package pl.javastart.task;
+
+public class Car extends  Vehicle {
+    private boolean isClimaTurnedOn;
+
+    public Car(String name, double tankCapacity, double averageConsumption, boolean isClimaTurnedOn) {
+        super(name, tankCapacity, averageConsumption);
+        this.isClimaTurnedOn = isClimaTurnedOn;
+    }
+
+    public double countRange(Vehicle vehicle, boolean isClimaTurnedOn) {
+        if (isClimaTurnedOn) {
+            return ((vehicle.getTankCapacity() / (vehicle.getAverageConsumption() + 0.8) * 100));
```

Comment:
Unikaj stosowania tzw. "magicznych liczb". Zastąp ją dobrze nazwaną stałą https://youtu.be/eY1li8xwX3M